dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -326,30 +326,28 @@ def __hash__(self):
 
 def find_provider_affected(changed_file: str, include_docs: bool) -> str | None:
     file_path = AIRFLOW_SOURCES_ROOT / changed_file
-    # is_relative_to is only available in Python 3.9 - we should simplify this check when we are Python 3.9+
-    for provider_root in (TESTS_PROVIDERS_ROOT, SYSTEM_TESTS_PROVIDERS_ROOT, AIRFLOW_PROVIDERS_NS_PACKAGE):
```

Comment:
Bug #1: TESTS were checked before SYSTEM_TESTS -> so system tests were never set as base provider's path because they are sub-folder of tests.